spec/business/book_creator_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe BookCreator do
+  it 'create a book by isbn' do
+    VCR.use_cassette('search-isbn') do
+      book = described_class.create_by_isbn('9781118084786')
+
+      expect(book).to be_a Book
+      expect(book.persisted?).to be true
+      expect(book.isbn).to eq '9781118084786'
+      expect(book.title).to eq 'Ruby on Rails For Dummies'
+      expect(book.authors).to eq 'Barry Burd'
+      expect(book.year_published).to eq 2011
```

Comment:
@rafaelrpbelo : https://gist.github.com/dmitryrck/14828a332b6274514bdf what do you think about?